components/homme/src/theta-l_kokkos/cxx/ForcingFunctor.hpp

```diff
@@ -236,7 +236,7 @@ class ForcingFunctor
     });
   }
 
-  void tracers_forcing (const Real dt, const int np1, const int np1_qdp, const bool adjustment, const MoistDry moisture) {
+  void tracers_forcing (const Real dt, const int np1, const int np1_qdp, const bool adjustment, const bool use_moisture) {
```

Comment:
I'm fine with this change, but was it done for a system reason? Or is it just something you want to do?

Reply:
moisture variable was used in some complicated way, through some type MoistDry::MOIST, MoistDry::DRY, i just made it boolean (and then int) instead.